.github/ISSUE_TEMPLATE/i-have-a-problem-with-rive-ios.md

```diff
@@ -7,36 +7,39 @@ assignees: ''
 
 ---
 
-<-- Thanks for using Rive!
+<!-- Thanks for using Rive!
 
 If something isn't working like you expected, please:
 - Make sure you've updated rive-ios to the latest version:
 			`pod update RiveRuntime`
 - Check out our documentation
 			https://rive.app/community/doc
 - Get help on our forums
-			https://rive.app/community/forums/home				
+			https://community.rive.app/
 - You can file a new feature request at
-			https://rive.app/community/forums/feature-requests/fsnweJpNLqIh
+			https://community.rive.app/c/feature-requests/
 
 If you think you found a bug or if you can't find what you're looking for 
 in our docs, fill out the template below.
 
 -->
 
 ### Description
-<-- A clear and concise description of what the problem is. -->
+
+<!-- A clear and concise description of what the problem is. -->
 
 ### Provide a Repro
-<-- 
+
+<!--
```

Comment:
Proper HTML comments, and adding a newline after Markdown headers to improve readability.